src/styles.css

```
@@ -516,6 +516,58 @@ a:focus-visible {
   display: none;
 }
 
+.strategy-adjustment-range {
+  --range-progress: 0%;
+  height: 8px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)